app/views/deals/_deal.html.erb

```diff
@@ -0,0 +1,23 @@
+<div id="<%= dom_id deal %>" class="flex flex-col shadow-lg gap-3 bg-white py-2 px-5 text-text-500 w-full mb-5">
+  <section class="flex justify-between items-start">
+  <div class="flex items-start sm:gap-5 gap-2 flex-col ">
+    <p class="font-proximabold text-3xl md:text-4xl text-text-300 font-medium">
+      <%= deal.name %>
+    </p>
+    <p class="font-proximalight text-xl text-text-200 font-medium">
+      <%= deal.created_at.to_formatted_s(:long_ordinal)   %>
+    </p>
+  </div>
+
+  <p class='font-proximalight text-2xl md:mr-10 font-bold'>
+    $<%= deal.amount %>
+  </p>
+  </section>
+
+  <% if action_name != "show" %>
+  <div class="flex gap-2 sm:gap-7">
+    <%= button_to 'Remove', category_deal_path(id:deal.id, category_id: params[:category_id]), method: :delete, class: "mt-2 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+    <%= link_to 'Edit', edit_deal_path(deal), class: "rounded-lg py-3 mt-2 px-5 bg-gray-100 inline-block font-medium" %>
+  </div>
```

Comment:
[OPTIONAL] The edit button for deals is not working. It throws an error.

![image](https://user-images.githubusercontent.com/38088818/185636362-aaaa23d0-ea7f-4197-9738-0ed78d57e843.png)

Kindly look into it or otherwise remove it.